packages/volto/src/components/theme/Login/Login.jsx

```diff
@@ -85,6 +86,9 @@ const Login = (props) => {
 
   const previousToken = usePrevious(token);
 
+  // Get user and place their data in the Redux store
+  useUser();
```

Comment:
This was my suggestion, but now I'm realizing it wasn't the best idea.

For one thing, this runs every time the login form renders, including before the user successfully logs in.

More importantly, it doesn't populate the redux store if the user reloads the page while already logged in.

I'll comment below with what I think we actually need to avoid sending multiple requests for the user at once.